vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -284,49 +323,53 @@ func (d *VolumeDriver) Create(r volume.Request) volume.Response {
 		// in which case we continue creating the file system.
 		fs.DevAttachWait(watcher, r.Name, device)
 	}
-	errMkfs := fs.Mkfs(mkfscmd, r.Name, device)
-	if errMkfs != nil {
-		log.WithFields(log.Fields{"name": r.Name,
-			"error": errMkfs}).Error("Create filesystem failed, removing the volume ")
-		errDetach := d.ops.Detach(r.Name, nil)
-		if errDetach != nil {
-			log.WithFields(log.Fields{"name": r.Name, "error": errDetach}).Warning("Detach volume failed ")
+	if err := fs.Mkfs(mkfscmd, r.Name, device); err != nil {
+		log.WithField("name", r.Name).WithError(err).Error(
+			"Create filesystem failed, removing the volume")
+		if err := d.ops.Detach(r.Name, nil); err != nil {
+			log.WithField("name", r.Name).WithError(err).Warn(
+				"Detach volume failed")
 		}
-		errRemove := d.ops.Remove(r.Name, nil)
-		if errRemove != nil {
-			log.WithFields(log.Fields{"name": r.Name, "error": errRemove}).Warning("Remove volume failed ")
+		if err := d.ops.Remove(r.Name, nil); err != nil {
+			log.WithField("name", r.Name).WithError(err).Warn(
+				"Remove volume failed")
 		}
-		return volume.Response{Err: errMkfs.Error()}
+		return volume.Response{Err: err.Error()}
 	}
 
-	errDetach := d.ops.Detach(r.Name, nil)
-	if errDetach != nil {
-		log.WithFields(log.Fields{"name": r.Name, "error": errDetach}).Error("Detach volume failed ")
-		return volume.Response{Err: errDetach.Error()}
+	if err := d.ops.Detach(r.Name, nil); err != nil {
+		log.WithField("name", r.Name).WithError(err).Error(
+			"Detach volume failed")
+		return volume.Response{Err: err.Error()}
 	}
 
-	log.WithFields(log.Fields{"name": r.Name,
-		"fstype": r.Options["fstype"]}).Info("Volume and filesystem created ")
+	log.WithFields(log.Fields{
+		"name":   r.Name,
+		"fstype": r.Options["fstype"],
+	}).Info("Volume and filesystem created")
 	return volume.Response{Err: ""}
 }
 
-// Remove - removes individual volume. Docker would call it only if is not using it anymore
+// Remove - removes individual volume. Docker would call it only if is not
```

Comment:
Hi @govint,

Please note this PR is closed. Also, you're commenting on something that predates this PR. All I did was wrap the line. The content did not change. If you feel like this should be fixed then I encourage you to fix it.